webclient/tests/http2/README.md

```diff
@@ -0,0 +1,3 @@
+To generate a new certificate:
+
+Refer to tests/integration/webserver/upgrade/src/main/resources/certificate_notes.md
```

Comment:
Copy the instructions instead of referring to another file.

Reply:
Done